src/ingest_llm_as/observability/langfuse_client.py

```
@@ -6,207 +6,120 @@
+            return
+
+        try:
+            generation = self.client.start_as_current_generation(
+                name=name,
+                model=model,
+                input=input_text,
+                output=output_text,
+                metadata=metadata,
+            )
+            return getattr(generation, "id", None)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)